spec/system/guessing_game_spec.rb

```diff
@@ -48,6 +48,10 @@
     expect(page).to have_text("Congratulations!")
   end
 
+  context "when at the exact position" do
```

Comment:
Remember to keep things tidy, remove code that does nothing, like this empty context.